lib/functions.php

```diff
@@ -0,0 +1,72 @@
+<?php defined('SESAM') or die('Direct access permitted');
+/*
+    Copyright (c) 2015 Oliver Lau <ola@ct.de>, Heise Medien GmbH & Co. KG
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+function sendResponse($data = array(), $success = true)
+{
+    $status = 'ok';
+
+    if (!$success) {
+        $status = 'error';
+    }
+
+    $data = array_merge(array(
+        'status' => $status
+    ), $data);
+
+    header('Content-Type: text/json');
+    $json = json_encode($data, JSON_UNESCAPED_SLASHES);
```

Comment:
`JSON_UNESCAPED_SLASHES` introduced in PHP 5.4. Will revert this line to

```
    $json = json_encode($data);
    echo str_replace("\/", "/", $json);
``` 

for backwards compatibility.